Assets/Scripts/Interactables/CollectableChip.cs

```diff
@@ -22,6 +24,8 @@ private void OnTriggerEnter(Collider other)
         if (!other.gameObject.TryGetComponent<PlayerManager>(out PlayerManager player))
             return;
         player.identity.UpdateChip(1);
-        Destroy(gameObject);
+        audioSource.Play();
+        GetComponent<Collider>().enabled = false;
+        chipModel.SetActive(false);
```

Comment:
You could still destroy the coin after the audio has played

Reply:
Ye, but I think the cost of running an IEnemurator waiting for the sound to finish playing is basicly the same as disabling the object like this, so I think this is the better solution for the brevity.

Reply:
Destroy(gameoObject, audioSource.clip.length)
(usikker på om det er clip)
Resolve hvis du ikke vil prøve